mne/viz/evoked.py

```diff
@@ -393,7 +394,8 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
 def plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
                 ylim=None, xlim='tight', proj=False, hline=None, units=None,
                 scalings=None, titles=None, axes=None, gfp=False,
-                window_title=None, spatial_colors=False):
+                window_title=None, spatial_colors=False,
+                set_tight_layout=True):
```

Comment:
How about only exposing this param for the private function below and using it in the code for ``joint_plot``? I don't think the public function needs this.